### PR TITLE
added tiles > autotiles proof

### PR DESCRIPTION
**Pull request by SeiferTim**

### CODE REVIEW

File: demos/drawing/drawline/source/PlayState.hx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			{type: FlxTweenType.PINGPONG});
+
+		FlxTween.linearMotion(spriteright, FlxG.width
+			- 10
+			- spriteright.width,
+			FlxG.height
+			- 10
+			- spriteright.height, FlxG.width
+			- 10
+			- spriteright.width, 10, 2, {
```
Comment (Geokureli) [inline demos/drawing/drawline/source/PlayState.hx:40]: this seems like an autoformatting error

**Review by Geokureli — CHANGES_REQUESTED** [review]
Seems weird to
split expressions right
in the middle. was
that on
purpose?

File: demos/drawing/drawline/source/PlayState.hx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+		FlxSpriteUtil.fill(line, 0);
+
+		FlxSpriteUtil.drawLine(line, spriteleft.x
+			+ spriteleft.origin.x,
+			spriteleft.y
+			+ spriteleft.origin.y, spriteright.x
+			+ spriteright.origin.x,
+			spriteright.y
+			+ spriteright.origin.y, {
```
[the remark this answers is not in the document]
Comment (SeiferTim) [reply demos/drawing/drawline/source/PlayState.hx]: It is being done by autoformatting... maxLinelength was too small. Change incoming.

**Review by Geokureli — APPROVED** [review]

**Comment by Geokureli** [comment]
Seems odd to use spaces, when everything else in flixel uses tabs, but honestly I don't mind either, and  I guess I don't see a need to enforce one.

**Pull request merged**